### DISCUSSION TITLE
Accepting RSS entries with empty title field

### DISCUSSION THREAD

**Discussion by arthurerb** [Flexget/Flexget#3652] [Q&A] [answered]
I'm trying to take entries from an RSS feed and pass them to a command via exec. This works fine if the RSS item has content in its title,  but the entries are skipped if the title is empty. I get a warning saying `Skipped %s RSS-entries without required information (title, link or enclosures)` if there are empty titles. The content I want to send is in the link field, so an empty title is not a worry for me.

Is there a way to either accept the entries with empty titles, or to force a default title instead?

**Comment by lostb1t** [comment, marked as answer]
You can use a different field for title that's always set like the link:

```
rss:
  url: <url>
  title: link
```

Using other_fields might also works but not sure

```
rss:
 other_fields:
   - "placeholder": title
```

**Reply by arthurerb** [reply]
Ah. I had been trying to do that with the modify plugin.
Putting it with the RSS URL works. Thanks!
